### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by Ashah15**

### CODE REVIEW

**Comment by mosaaleb** [comment]
## Under review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<nav class="top-nav">
+		<ul>
+			<li><i class="fa fa-apple" aria-hidden="true"></i></li>
+			<li>Store</li>
+			<li>Mac</li>
```
Comment (mosaaleb) [inline index.html:15]: All the `<li>` items inside `nav` needs to be links.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			<li>iTunes</li>
+			<li>Support</li>
+			<li class="searchbox-con"><input type="search" name="search" class="searchbox"></li>
+		</ul>
+	</nav>
+
+	<main>
+		<section class="top-header">
+			<p class="top-header-title">What will your verse be?</p>
+			<p class="top-header-subtitle">Everyone has something to share. See how people are using iPad in their unique ways to add to the worlds story.</p>
```
Comment (mosaaleb) [inline index.html]: All the text inside `top-header` section are also clickable links:
- Each link is an `<a>` tag, with a `<div>` tag inside it.
- The `<div>` tag  inside the `<a>` tag should use an image as background -- [ ] The text on the original page is actually an image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			<p class="top-header-link">Watch the "Your Verse" film and explore the stories <i class="fa fa-play-circle-o" aria-hidden="true"></i></p>
+		</section>
+		<section class="promo-pages">
+			<div class="col"><img src="images/promo_iphone5s.jpg" class="img-responsive" alt="iphone 5s promo"></div>
+			<div class="col"><img src="images/promo_iphone5c.jpg" class="img-responsive" alt="iphone 5c promo"></div>
```
Comment (mosaaleb) [inline index.html]: The images inside the `promo-pages` section are also clickable links.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @Ashah15 :wave:

### Amazing work :clap:. Your code is organized and easy to read. The UI is perfect. You have already handled most of the requirements, still there is some work to do before moving on. See my comments below.

- [ ] Also, let's make sure that our style guidelines are ok by using [stickler](https://github.com/microverseinc/linters-config/tree/master/css). It's very easy to use, just follow the link.

Please submit a new code review once you finish these minor requirements.

#### Happy coding :computer:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+		 <li><a href=""><span>iTunes</span></a></li>
+		 <li><a href=""><span>Support</span></a></li>
+
+      <li class="searchbox-con"><i class="fa fa-search"></i><input type="search" name="search" class="searchbox"></li>
+		</ul>
+	</nav>
+
+	<main>
+		<section class="top-header">
+			<div class="hero-section">
```
Comment (Oghenebrume50) [inline index.html]: Hi, you are making great progress but you did not correct this point from the previous review you got, please pass the images as background images usng CSS
- [ ] Each link is an <a> tag, with a <div> tag inside it.
- [ ] The <div> tag inside the <a> tag should use an image as background, The text on the original page is actually an image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+				<a href=""><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" alt=""	/></a>
+
+        <a href=""><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" alt="" /></a>
+
+				<a href=""><img	src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" alt=""/></a>
+					</div>
+		</section>
+		<section class="promo-pages">
```
Comment (Oghenebrume50) [inline index.html]: Do the same for the images in this container also
- [ ] Each link is an <a> tag, with a <div> tag inside it.
- [ ] The <div> tag inside the <a> tag should use an image as background, The text on the original page is actually an image

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+		</section>
+		
+	</main>
+
+	<footer>
+		<section class="promo-pages">
+			<div class="col first-promo-img"></div>
+			<div class="col second-promo-img"></div>
+			<div class="col third-promo-img"></div>
+			<div class="col fourth-promo-img"></div>
```
Comment (Oghenebrume50) [inline index.html:40]: Hi this is good every works fine althought these divs  here are suppose to be links that is wrapped within an a tag like I stated in the last review. You could correct that and submit this project. :+1:  goodluck on your next project :+1:

**Review by Oghenebrume50 — APPROVED** [review]

**Pull request merged**